src/components/notifications/Webpush.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <button
+        class="mb-3"
+        type="button"
+        :class="[
+            'btn',
+            canRegister ? 'btn-primary' : 'btn-danger'
+        ]"
+        :disabled="!btnEnabled"
+        @click="registerWebpush"
+    >
+        <div v-if="processing" class="spinner-border spinner-border-sm me-1"></div>
+        <span v-else-if="$parent.notification.subscription" class="me-1">✓</span>
+        {{ btnText }}
+    </button>
+
+    <div class="mb-3 form-text">
+        <a href="TODO" target="_blank">{{ $t("documentationOf", ["Webpush"]) }}</a>
+    </div>
+</template>
+
+<script>
+export default {
+    data() {
+        return {
+            //store subscription info
+            btnEnabled: false,
+            btnText: "",
+            processing: false,
+            //determines if browser supports service worker
+            canRegister: false,
```

Comment:
Please rename this as such 

```suggestion
            browserSupportsServiceWorkers: false,
```